docs/quickstarts/nextjs.mdx

```diff
@@ -46,38 +48,40 @@ pnpm add @clerk/nextjs
 
 </CodeBlockTabs>
 
-### Set environment keys
+### Set your environment keys
 
-In your Next.js project's root folder, you may have an `.env.local` file alongside `package.json` and other configuration files. If you don't see it, create it.
+1. If you don't have a `.env.local` file in the root directory of your Next.js project, create one now.
+2. Find your secret keys in your Clerk Dashboard:
+    - Navigate to your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys) to find your secret keys.
+    - Select your application, then select **🔑 API Keys** in the sidebar menu.
+    - You can copy your keys from the **Quick Copy** section.
+3. Add your keys to your `.env.local` file. The final result should look as follows:

```

Comment:
Removing "to find your secret keys" as it seems redundant - we say this in the line above.

```suggestion
2. Find your secret keys in your Clerk Dashboard:
    - Navigate to your [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys).
    - Select your application, then select **🔑 API Keys** in the sidebar menu.
    - You can copy your keys from the **Quick Copy** section.
3. Add your keys to your `.env.local` file. 

The final result should look as follows:

```